src/tray/trayicon/trayicon.cpp

```diff
@@ -95,16 +95,41 @@ QMenu* TrayIcon::createTrayIconMenu()
     connect(quitAction, SIGNAL(triggered()), this, SLOT(exitEvent()));
 
     // Setting system tray's icon menu...
+    autoProfile = new QAction(tr("Auto profile"), this);
+    autoProfile -> setCheckable(true);
+    autoProfile -> setChecked(tunedManager -> GetProfileModeAuto());
+    if (autoProfile->isChecked())
+        autoProfile->setDisabled(true);
```

Comment:
It always should be disabled IMO.

Reply:
If it's initially unchecked (i.e. profile was selected manually before launch), it should be checkable.

Reply:
OK. ACK.